04-materials/05-developing-with-ai.md

```diff
@@ -12,3 +12,443 @@ After this module, you will have:
+- **Download:** [cursor.com](https://cursor.com/)
+
+#### 2. 💻 **Claude Code**
+- **What it is:** Command-line interface for Claude, optimized for coding workflows
+- **LLM Options:**
+  - Requires Claude subscription or Anthropic API key. Can also work through cloud providers, like Amazon Bedrock
+  - Works with Opus 4.1, Sonnet 4.5, Haiku 4.5, and other Claude models
+- **Best for:** CLI warriors who live in the terminal
+- **Alternatives:** [Gemini CLI](https://github.com/google-gemini/gemini-cli) (free tier available), [Cline](https://github.com/cline/cline) (VS Code extension with CLI mode, free), [Continue](https://github.com/continuedev/continue) (IDE/terminal/CI agent, free), [Plandex](https://github.com/plandex-ai/plandex) (designed for large projects), [aichat](https://github.com/sigoden/aichat) (all-in-one LLM CLI), [GitHub Copilot CLI](https://github.com/github/copilot-cli), [Aider](https://github.com/Aider-AI/aider) (Git-integrated, open-source), [Google Jules](https://jules.google/) (async background agent, beta)
+- **Install:** Requires Node.js, then `npm install --global @anthropic/claude-code`
+
+:::{note} Further Reading
+:class: dropdown
+- [Simon Willison's blog on AI-assisted programming](https://simonwillison.net/tags/ai-assisted-programming/) — Practical insights from a prolific developer
+- ["Not all AI-assisted programming is vibe coding (but vibe coding rocks)"](https://simonwillison.net/2025/Mar/19/vibe-coding/) — Understanding different AI coding styles
+- [Claude Sonnet 4.5 review](https://simonwillison.net/2025/Sep/29/claude-sonnet-4-5/) — Simon calls it "the best coding model in the world"
+- [Parallel Coding Agents](https://simonwillison.net/2025/Oct/5/parallel-coding-agents/) — Simon's October 2025 workflow with Claude Sonnet 4.5
+- [Simon Willison's Model Recommendations](https://simonwillison.net/2025/Oct/) — Qwen3 and GLM-4.5 for open-source coding; Chinese models often outperform Western alternatives
+- [Ollama Documentation](https://ollama.com/) — Self-hosting open-source models
+- [LM Studio](https://lmstudio.ai/) — GUI for running local LLMs
+:::
+
+## 🏁 Getting started
+
+### 📦 Repo
+For this module, we will start with an existing extension that we built in chapter 2. If you are not caught up or just joining us for the afternoon session, please grab a reference implementation from [our demo repository](https://github.com/mfisher87/jupytercon2025-developingextensions-demo).
+
+In {doc}`02-anatomy-of-extensions`, we started off by cloning an official [JupyterLab extension template](https://github.com/jupyterlab/extension-template).
```

Comment:
```suggestion
In [](./02-anatomy-of-extensions.md), we started off by cloning an official [JupyterLab extension template](https://github.com/jupyterlab/extension-template).
```

Use of `{doc}` isn't recommended and produces warnings, this is the equivalent :point_up: